bin/i18n/sync-in.rb

```diff
@@ -248,23 +285,19 @@ def localize_level_content
     end
   end
 
-  File.open(File.join(I18N_SOURCE_DIR, "dashboard/block_categories.yml"), 'w') do |file|
-    # Format strings for consumption by the rails i18n engine
-    formatted_data = {
-      "en" => {
-        "data" => {
-          "block_categories" => block_category_strings.sort.to_h
-        }
-      }
-    }
-    file.write(I18nScriptUtils.to_crowdin_yaml(formatted_data))
-  end
-  File.open(File.join(I18N_SOURCE_DIR, "dashboard/progressions.yml"), 'w') do |file|
+  write_to_yml(block_categories, block_category_strings)
+  write_to_yml(progressions, progression_strings)
+  write_to_yml(variable_names, variable_strings)
+  write_to_yml(parameter_names, parameter_strings)
+end
+
+def write_to_yml(block_type, block_strings)
```

Comment:
I don't think anything here is block-specific, is it? Should probably be something like

```suggestion
def write_to_yml(type, strings)
```